dev/bin/interaction/environment-example

```diff
@@ -2,8 +2,14 @@
 # It can be used to configure required variables for your local environment
 # without the need of exporting them every time, like offline tokens etc.
 # If you want to use it, make a copy of this named "environment" in this same folder.
+# The dev and stable environment use sso.redhat.com. So if you want to target such environments, set TARGET_ENVIRONMENT=remote and the
+# variable OPENSHIFT_OFFLINE_TOKEN accordingly (retrieve the token from https://console.redhat.com/openshift/token).
```

Comment:
Don't we have a script also for this? I know it is a single command but it helps people :)